### PR TITLE
Prepare an example of python notebbok with fluree repository

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
 behave~=1.2.6
 #notebook~=7.2.0
 #linkml~=1.7.10
+aioflureedb[domainapi]~=0.8.3
+jupyter~=1.0.0
```
Comment (schivmeister) [inline requirements.txt:5]: let's not depend on this metapackage if not every notebook feature is needed (use the notebook one instead, and we can move these to a separate file as they are optional)

**Review by schivmeister — CHANGES_REQUESTED** [review]
@duprijil thanks for this, great work. i have left some minor comments, check them out if you can. i might also make some changes and repush with rebase, will let you know if so.

File: activate_venv.sh
[the remark this answers is not in the document]
Comment (duprijil) [reply activate_venv.sh:1]: deleted

File: data_centric_app_demo/models/person.py
[the remark this answers is not in the document]
Comment (duprijil) [reply data_centric_app_demo/models/person.py:1]: For now, I added an example to see how it is generated. Before inject generated class, we need to have a discussion about that.

File: notebooks/linkml-fluree.ipynb [hunk 2]
```diff
@@ -170,7 +215,11 @@
    "cell_type": "code",
    "execution_count": 398,
    "id": "3f3cdfa3-d6ee-4139-b67a-3bb4a3d0da94",
-   "metadata": {},
+   "metadata": {
```
[the remark this answers is not in the document]
Comment (duprijil) [reply notebooks/linkml-fluree.ipynb:154]: Metadata comes from output. A good practice is to push notebooks to repository without output.
So I propose to clear output and restart kernel before pushing so we will not have changes in notebook json metadata, like execution counts or output metadata.

File: notebooks/linkml-fluree.ipynb [hunk 3]
```diff
@@ -44,24 +56,29 @@
...
-      "    attributes:\n",
-      "      persons:\n",
-      "        multivalued: true\n",
-      "        inlined_as_list: true\n",
-      "        range: Person\n",
-      "      companies:\n",
-      "        multivalued: true\n",
-      "        inlined_as_list: true\n",
-      "        range: Company\n"
+      "/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)\r\n",
```
[the remark this answers is not in the document]
Comment (duprijil) [reply notebooks/linkml-fluree.ipynb]: See previous comment

File: requirements.txt (hunk 1 above)
Comment (duprijil) [reply requirements.txt:4]: Done

**Review by schivmeister — APPROVED** [review]

**Comment by schivmeister** [comment]
@duprijil done, all good

**Pull request merged**